koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -152,7 +152,7 @@ def process(self, parquet_base_filename, daily_data_frames):
             if self.has_enabled_ocp_labels:
                 LOG.info("Getting matching tags from Postgres.")
                 matched_tags = self.db_accessor.get_openshift_on_cloud_matched_tags(self.bill_id, report_period_id)
-                if not matched_tags:
+                if not matched_tags or matched_tags is not None:
```

Comment:
I'm confused by this logic. Are we wanting to run this when there _are_ matched tags?

Reply:
I can't figure out when this if statement is not True.
```
>>> matched_tags = None
>>> not matched_tags
True
>>> matched_tags is not None
False
>>> matched_tags = True
>>> not matched_tags
False
>>> matched_tags is not None
True
>>> tags = []
>>> not tags
True
>>> tags is not None
True
>>> tags = ['something']
>>> not tags
False
>>> tags is not None
True
>>> 
```

Reply:
ahhhhh yes! Thank you @maskarb 

It is confusing because it is suppose to be an `and` condition and not an  `or`.  I had to go back and look at my test file 😂 

```
(koku) cmyers@cmyers1-mac koku % cat test.py
matched = []
if not matched and matched is not None:
    print(matched)

matched = None
if not matched and matched is not None:
    print(matched)

matched = "ABC"
if not matched and matched is not None:
    print(matched)%
(koku) cmyers@cmyers1-mac koku % python3 test.py
[]
 
```

Essentially, we only want to run the Trino tag matching if it is an empty `[]`. Been staying up to late man 😂 

But now that you mention it, returning a None is a bit confusing, I am gonna replace it with the string `skip`. Thanks for the extra set of eyes.

Reply:
I was debating returning a `None` or the string `skip` in my head during this and I decided to return None, but forgot to change the if condition to match. :smh: